runtime/caml/compatibility.h

```diff
@@ -23,11 +23,25 @@
 #define caml_stat_heap_size Bsize_wsize(caml_stat_heap_wsz)
 
 /* global variables moved to Caml_state in 4.10.0 */
+#define caml_stat_top_heap_wsz (Caml_state_field(stat_top_heap_wsz))
+#define caml_stat_heap_wsz (Caml_state_field(stat_heap_wsz))
 #define caml_young_start (Caml_state_field(young_start))
 #define caml_young_end (Caml_state_field(young_end))
 #define caml_young_ptr (Caml_state_field(young_ptr))
 #define caml_young_limit (Caml_state_field(young_limit))
 #define caml_local_roots (Caml_state_field(local_roots))
+#define caml_backtrace_active (Caml_state_field(backtrace_active))
+#define caml_backtrace_pos (Caml_state_field(backtrace_pos))
+#define caml_backtrace_buffer (Caml_state_field(backtrace_buffer))
+#define caml_backtrace_last_exn (Caml_state_field(backtrace_last_exn))
+#define caml_compare_unordered (Caml_state_field(compare_unordered))
```

Comment:
I think this `#define` conflicts with the declaration in `<caml/custom.h>` line 69:
```
CAMLextern int caml_compare_unordered;
```
That would explain some of the CI failures.

Could you please remove this declaration from `custom.h` so that we can test further?

Reply:
Done.